### PR TITLE
fix #948

### LINKED ISSUE #948: add "notesonly" argument to \gresetshiftaftermora
This argument would ignore punctum mora in hozirontal spacing between notes only, not between bars.

### PR DESCRIPTION
**Pull request by eroux**

### CODE REVIEW

File: tex/gregoriotex-signs.tex [hunk 1]
```diff
@@ -1620,20 +1620,23 @@
...
-    {\IfStrEq{#1}{barsonly}%
-      {\gre@count@shiftaftermora=3}%
-      {\IfStrEq{#1}{barsnotextonly}%
-        {\gre@count@shiftaftermora=2}%
-        {\IfStrEq{#1}{barsinsideonly}%
-          {\gre@count@shiftaftermora=1}%
-          {\IfStrEq{#1}{never}%
-            {\gre@count@shiftaftermora=0}%
-            {\gre@error{Unrecognized option "#1" in gresetshiftaftermora , use 'always', 'barsonly', 'barsnotextonly', 'barsinsideonly' or 'never'}}%
+    {\gre@count@shiftaftermora=5}%
```
Comment (henryso) [inline tex/gregoriotex-signs.tex]: Not that it really matters, but I find that `\IfStrEqCase` is more readable when there are more than a few cases.

**Comment by henryso** [comment]
Looks OK to be (though I'm not the best person to review TeX).

File: tex/gregoriotex-signs.tex (hunk 1 above)
Comment (eroux) [inline tex/gregoriotex-signs.tex]: you're right... but the clock is ticking, we'll clean this in the future (maybe)

**Comment by eroux** [comment]
Thanks!

**Pull request merged**